portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import com.google.sps.utils.CommentDataStore;
+import com.google.sps.utils.ServletUtils;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.List;
+
+/** Servlet that handles posting list of comments. */
+@WebServlet("/list-comment")
+public class ListCommentServlet extends HttpServlet {
+
+    private CommentDataStore commentDataStore;
+
+    @Override
+    public void init() {
+        this.commentDataStore = new CommentDataStore(DatastoreServiceFactory.getDatastoreService());
+    }
+
+    private static final String PARAM_NAME_QUANTITY = "quantity";
```

Comment:
this and the one below are not used. can you remove them?

Reply:
Actually it is used haha